packages/editor/src/lib/editor/Editor.ts

```diff
@@ -4623,7 +4605,6 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 */
 	@computed getCurrentPageRenderingShapesSorted(): TLShape[] {
 		return this.getRenderingShapes()
-			.filter(({ isCulled }) => !isCulled)
```

Comment:
I guess this could make the `getShapeAtPoint` a bit slower, but on my rbush PR I do improve it significantly.